packages/dds/tree/src/core/rebase/utils.ts

```diff
@@ -74,6 +89,18 @@ export interface BranchRebaseResult<TChange> {
 	 * Details about how the commits on the source branch changed
 	 */
 	readonly commits: RebasedCommits<TChange>;
+	/**
+	 * Telemetry properties for the rebase operation.
+	 */
+	readonly telemetryProperties?: TelemetryProperties;
```

Comment:
Shouldn't this be expected always?
```suggestion
	readonly telemetryProperties: TelemetryProperties;
```
```suggestion
	readonly telemetryProperties?: TelemetryProperties;
```

Reply:
Thinking about it, you're right. I was picturing maybe we could pass an optional boolean flag (maybe something like `getTelemetry` with default value of False) and compute the telemetry if passed as True. 

But we are computing the values anyway in every rebases scenarios, so yes it should be expected. Thanks 😄 !